packages/byods/src/Errors/types.ts

```diff
@@ -0,0 +1,60 @@
+import {IMetaContext} from '../Logger/types';
+
+export type ErrorMessage = string;
+
+export enum ERROR_LAYER {
+  CALL_CONTROL = 'call_control',
+  MEDIA = 'media',
+}
+
+export enum ERROR_TYPE {
+  CALL_ERROR = 'call_error',
```

Comment:
Don't think we need `CALL_ERROR` here - correct me if I'm wrong.

Reply:
@adhmenon Updated...!